inputs/values/ref_panel_1kg.json

```diff
@@ -2236,6 +2238,7 @@
     "name": "ref_panel_1kg",
     "outlier_cutoff_table": "gs://gatk-sv-resources-public/hg38/v0/sv-resources/ref-panel/1KG/v1/module03_outlier_cutoff_table.tsv",
     "ped_file": "gs://gcp-public-data--broad-references/hg38/v0/sv-resources/ref-panel/1KG/v1/ped/1kg_ref_panel_v1.ped",
+    "ploidy_table": "gs://gatk-sv-ref-panel-1kg/outputs/ClusterBatch/mw-concordance/ref_panel_1kg.ploidy.FEMALE_chrY_1.tsv",
```

Comment:
Should open a ticket to track changes needed to be able to repopulate this JSON with create_test_batch.py including outputting this file from the Batch pipeline, or keeping this value along with the new batch's metadata. I think some other recent changes might need to sync up with the create_test_batch.py script too (ex. handling tarballs of std vcfs vs. the file lists used in the single-sample pipeline) so it would be good to keep track of those

Reply:
Done